pkg/apis/cartographer/v1alpha1/workload.go

```diff
@@ -38,6 +38,11 @@ const (
 	ResourceRealizerBuilderErrorResourcesSubmittedReason = "ResourceRealizerBuilderError"
 )
 
+const (
+	ResourceReady     = "Ready"
```

Comment:
For condition const, use name `Condition` prefix in the name like `ConditionResourceReady`. Same for both conditions.

Reply:
FYI, I used it as it is in Carto source code.